lib/external_interaction/rubix/rubix_external_rest_api.dart

```diff
@@ -25,4 +28,26 @@ Future<Response> handleTransactionRequest(Request request) async {
   return Response.ok('Transaction request received');
 }
 
+Future<bool> externalApi({required String did,required String peerId,required String orgName,required String callBackUrl,required Token orgAccessToken })async {
```

Comment:
This function should be named fireAuthCallback or something similar. It 
doesn't need orgName
did and peerId need not be separate

Reply:
Ok will make these changes and put a new commit.